### ISSUE TITLE
RFC: New function for operating with core.Namespace

### ISSUE THREAD

**Issue by marcin-krolik** [intelsdi-x/snap#884]
When dealing with more then one static element following dynamic element one need to "spam" ```AddStaticElement(element string)``` function. 
I propose to introduce new function to add possibility to add arbitrary number of static elements

```go
func (n Namespace) AddStaticElements(values ...string) Namespace {
    for _, value := range values {
        n = append(n, NamespaceElement{Value: value})
    }
    return n
}
```

**Comment by jcooklin** [comment]
👍

**Comment by tiffanyfj** [comment]
Fixed by #885
